.ci/coverage.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# this is in a separate script from `.travis.yml` so we can set this option.
+shopt -s extglob
+
+wget https://github.com/SimonKagstrom/kcov/archive/master.tar.gz &&
+tar xzf master.tar.gz &&
+cd kcov-master &&
+mkdir build &&
+cd build &&
+cmake .. &&
+make &&
+make install DESTDIR=../tmp &&
+cd ../..
+
+if [ $TRAVIS_RUST_VERSION = nightly ]; then
+    ./kcov-master/tmp/usr/local/bin/kcov --exclude-pattern=/.cargo --coveralls-id=$TRAVIS_JOB_ID --verify  target/kcov target/debug/*-*[!.]? &&
```

Comment:
be careful to quote "$TRAVIS_JOB_ID" so it can't be used to inject nefarious things ;)

(PSA `shellcheck` is a great tool that will complain about this sort of thing

Reply:
can I `brew install shellcheck`?